src/System.Drawing.Common/tests/Configurations.props

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
Why is this needed?

Reply:
So that TargetsWindows variable is set. If not it would target AnyOS from AFAIK.

Reply:
But you're checking for != TargetsWindows.  Adding this netcoreapp-Windows_NT line results in TargetsUnix being set?

Reply:
Whenever AnyOS is set TargetsWindows is not set to true, it would be empty, so TargetsWindows != true will be a true statement and the properties will be set only for when OSGroup is AnyOS.

Reply:
I'm probably just not understanding...

For the netcoreapp-Windows_NT configuration, it'll only run on Windows, right?  So for that run, TargetsWindows will be true, correct?

Why is it that adding this configuration affects how the other configurations are processed?  It sounds like you're saying that for the other netcoreapp and netfx configuration, whether or not this netcoreapp-Windows_NT line is here affects the value of TargetsWindows in those runs?

Reply:
> For the netcoreapp-Windows_NT configuration, it'll only be compiled for and run on Windows, right? So for that run, TargetsWindows will be true, correct?

Correct.

>Why is it that adding this configuration affects how the other configurations are processed? It sounds like you're saying that for the other netcoreapp and netfx configuration, whether or not this netcoreapp-Windows_NT line is here affects the value of TargetsWindows in those builds?

When we run in Unix it will pick the netcoreapp configuration and OSGroup will be AnyOS and then TargetsWindows will be != true.

> I would have expected you'd have added a netcoreapp-Unix configuration, and then in the build enabled those settings when TargetUnix was true.

This would work also, I'm just doing the inverse, I will do it this way so it's less confusing.

Reply:
> When we run in Unix it will pick the netcoreapp configuration and OSGroup will be AnyOS and then TargetsWindows will be != true.

But won't we also run the netcoreapp and netfx configurations on Windows?

Reply:
> But won't we also run the netcoreapp and netfx configurations on Windows?

Yes, but when there is a OS specific configuration is the preferred configuration that the build system takes if it is eligible (best configuration) with the running OS. So if there is:

```
netcoreapp-Windows_NT;
netcoreapp;
```

When TargetGroup=netcoreapp if running in windows it will take the `netcoreapp-Windows_NT` configuration, but when running in any other OS it will take `netcoreapp` and the OSGroup will be `AnyOS`. 

The same applies for netfx but netfx can only run in windows so that is why we can leave the OS-less configuration.

However I will change it to what you proposed above so that it is not confusing.

Reply:
> Yes, but when there is a OS specific configuration is the preferred configuration that the build system takes if it is eligible (best configuration) with the running OS

I see.  That is confusing :)

Reply:
> I see. That is confusing :)
I know it is confusing but it makes sense to behave that way, if I specify a OS specific I want that configuration to be picked up whenever I'm running in that OS :)

> Thanks for clarifying, though.

No problem :)